problemsets-mat223/modules/module6.tex

```diff
@@ -164,8 +164,30 @@
 	\label{EXLINEBASIS}
 	Let $\ell=\Span\Set*{\mat{1\\2},\mat{-2\\-4}, \mat{1/2\\1}}$. Find
 	two different bases for $\ell$.
-
-	XXX Finish
+	
+	We are looking for a set of linearly independent vectors which spans $\ell$.
+	Notice that $\mat{1\\2}$ is linearly dependent of $\mat{-2\\-4}$
+	and linearly dependent of $\mat{1/2\\1}$.
+	This tells us that if a basis for $\ell$ contains one of these three vectors,
+	it cannot contain any of the other two.
+    Furthermore, we can see that
+    \[
+		\Span\Set*{\mat{1\\2}} =
+		\Span\Set*{\mat{-2\\-4}} =
+		\Span\Set*{\mat{1/2\\1}} =
+		\Span\Set*{\mat{1\\2},\mat{-2\\-4}, \mat{1/2\\1}} =
+		\ell
+    \]
+    due to the linear dependence among $\mat{1\\2}$, $\mat{-2\\-4}$, and $\mat{1/2\\1}$. 
```

Comment:
Eliminate what follows. You can say "Because $\Set*{..}$ is linearly independent, and spans $\ell$, we have that $\Set...$ is a basis for $\ell$. Similarly, ... is another basis for $\ell$.